test/cosmos/multinode.sh

```diff
@@ -218,9 +221,9 @@ if [[ ! -d "$hdir" ]]; then
 	perl -i -pe 's|"localhost:6060"|"localhost:6062"|g' $n2cfg
 	perl -i -pe 's|"tcp://0.0.0.0:10337"|"tcp://0.0.0.0:12337"|g' $n2app
 	perl -i -pe 's|"0.0.0.0:1317"|"0.0.0.0:1517"|g' $n2app
-	perl -i -pe 's|"0.0.0.0:9090"|"0.0.0.0:9092"|g' $n2app
+	perl -i -pe 's|minimum-gas-prices = ""|minimum-gas-prices = "'$minimumGasPrice'"|g' $n2app
 	perl -i -pe 's|allow_duplicate_ip = false|allow_duplicate_ip = true|g' $n2cfg
-	perl -i -pe 's|log_level = "info"|log_level = "'$LOG_LEVEL'"|g' $n2cfg
+	perl -i -pe 's|log_level = "debug"|log_level = "'$LOG_LEVEL'"|g' $n2cfg
```

Comment:
do we really want debug level?

Reply:
nop, my bad

Reply:
undo at https://github.com/umee-network/peggo/pull/343/commits/11a75ff97dfff42b95a976fd3b4a3dbaf039cf04